src/plugins/workspace/public/utils.ts

```diff
@@ -99,3 +107,16 @@ export function isAppAccessibleInWorkspace(app: App, workspace: WorkspaceObject)
   }
   return false;
 }
+
+export const filterWorkspaceConfigurableApps = (applications: PublicAppInfo[]) => {
+  const visibleApplications = applications.filter(({ navLinkStatus, chromeless, category, id }) => {
+    return (
```

Comment:
Shall we add some unit tests here to cover this line?

Reply:
Yes, there will be a followup PR from @yubonluo to cover this.